README.adoc

```diff
@@ -144,14 +144,23 @@ HiveMQ Community Edition offers an embedded mode and a programmatic API for inte
 
 === Dependency
 
+IMPORTANT: Until HiveMQ Community Edition 2020.3 is released embedded mode is only available from link:https://jitpack.io/#hivemq/hivemq-community-edition[jitpack.io].
+
 ==== Gradle
 
 If you use Gradle, include the following code in your `build.gradle` file.
 
 [source,groovy]
 ----
+	allprojects {
```

Comment:
allprojects is not necessary, please remove

Reply:
done